boa_engine/src/vm/opcode/meta/mod.rs

```diff
@@ -0,0 +1,94 @@
+use std::unreachable;
+
+use crate::{
+    module::ModuleKind,
+    vm::{opcode::Operation, ActiveRunnable, CompletionType},
+    Context, JsObject, JsResult, JsValue,
+};
+
+/// `NewTarget` implements the Opcode Operation for `Opcode::NewTarget`
+///
+/// Operation:
+///  - Push the current new target to the stack.
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct NewTarget;
+
+impl Operation for NewTarget {
+    const NAME: &'static str = "NewTarget";
+    const INSTRUCTION: &'static str = "INST - NewTarget";
+
+    fn execute(context: &mut Context<'_>) -> JsResult<CompletionType> {
+        let new_target = if let Some(new_target) = context
+            .vm
+            .environments
+            .get_this_environment()
+            .as_function()
+            .and_then(|env| env.slots().new_target().cloned())
+        {
+            new_target.into()
+        } else {
+            JsValue::undefined()
+        };
+        context.vm.push(new_target);
+        Ok(CompletionType::Normal)
+    }
+}
+
+/// `ImportMeta` implements the Opcode Operation for `Opcode::ImportMeta`
+///
+/// Operation:
+///  - Push the current `import.meta` to the stack
+///
+/// Operands:
+///
+/// Stack:
+///  - **=>** `import.meta`
```

Comment:
```suggestion
```
Maybe we should omit this, to be consistent with other Opcode implementations. This info is also on the `Opcode` enum too.